ykman/cli/__main__.py

```diff
@@ -98,8 +98,10 @@ def _run_cmd_for_serial(ctx, cmd, transports, serial):
                     ctx.fail("Command '{}' is not supported by this device."
                              .format(cmd))
         except FailedOpeningDeviceException:
-            ctx.fail('Failed connecting to a YubiKey with serial: {}'
-                     .format(serial))
+            ctx.fail(
+                'Failed connecting to a YubiKey with serial: {}. \
+                        Make sure you have the correct permissions'.format(
```

Comment:
How about "Make sure the application has the necessary permissions" instead? Can we provide any hint to what those permissions are (maybe just a link to some documentation)?

Reply:
The first suggested change I can do, rephrase it that its the app that may need permissions. But I don't want to add links to online documentation or detect the operating system, since it's mostly guessing anyway.